core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -3170,7 +3075,7 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
   }
 
   @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
+  @ValueSource(strings = Array("kraft"))
```

Comment:
There are still some zk code left in `testIncrementalAlterConfigsForLog4jLogLevels`, I didn't remove them as this test seems totally broken and we didn't enable the test for a long time. To fix this we might need to look into the related JIRA [KAFKA-8779](https://issues.apache.org/jira/browse/KAFKA-8779) again.